packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -98,10 +128,33 @@ function StepOne({ kycStatus }: StepOneProps) {
           <Text style={styles.description}>{t('linkBankAccountScreen.pending.description')}</Text>
         </View>
       )
-    case KycStatus.NotCreated:
-    case KycStatus.Created:
+    case KycStatus.Completed:
```

Comment:
I thought that "Completed" meant that the user completed the KYC but it was not yet approved, which was why we previously showed the "Verification Pending" screen. Why are we now showing the "Verification Complete" screen?

Reply:
you can see the description of various persona lifecycle states here: https://docs.withpersona.com/docs/models-lifecycle
And also i tested this, if the inquiry was successful without needing to be manually reviewed, `completed` will be the updated state